### DISCUSSION TITLE
儿童头颅CT数据集征集

### DISCUSSION THREAD

**Discussion by sam898** [linhandev/dataset#11] [数据集收集]
请填写数据集基本信息，*为必填项，其他内容请尽量填写．

注：开 Issue 后数据集成功并入项目，您的Github ID不会作为项目合作者在项目主页展示，我们会在项目末尾致谢．如果您希望作为项目合作者被展示请[提交PR](https://github.com/linhandev/dataset/pulls)．

数据集名称*：
标注内容*（标注的器官/病灶）：	
LICENSE*（关于数据集使用和传播的协议，常用LISENCE请写名字，如果是Custome LICENSE请写网址）：
下载方式*（下载网址或介绍下载方式的网页）：
类型（分类/分割/检测/...）：	
模态（CT/MR/CXR/...）：	
数量（扫描总数/图片张数）：	
标签格式：	
文件格式：

**Comment by linhandev** [comment]
有具体的数据集吗

**Comment by sam898** [comment]
我在网上只查到几张图片，不知贵司有没有儿童头颅CT的数据


| |
珊巴嘎
|
|
***@***.***
|
签名由网易邮箱大师定制
在2021年5月11日 18:25，Lin ***@***.***> 写道：

有具体的数据集吗

**Comment by linhandev** [comment]
我这个是个人项目，儿童头颅数据集之前我没见过，你那些图可以发一下，我找找
